src/logger/logger.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
```

Comment:
please move this file to src/middlewares
And rename this middleware to LogIpMiddleware because we only log IP